src/Main.java

```diff
@@ -1,5 +1,98 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
+        // Задача 1
+        System.out.println(" Задача 1");
+        int[] weights = new int[]{1, 2, 3};
+        float[] numbers = {1.57f, 7.654f, 9.986f};
+        int[] others = {1235, 987654, 654654, 321987};
+
+        // Задача 2
+        System.out.println(" Задача 2");
+
+        for (int i = 0; i < weights.length; i++) {
+            if (i == weights.length - 1) {
+                System.out.print(weights[i]);
```

Comment:
```suggestion
                System.out.println(weights[i]);
```
можно здесь сделать println, чтобы после цикла не писать его